Squadrosu.Framework/Board.cs

```diff
@@ -11,21 +14,51 @@ namespace Squadrosu.Framework;
 /// </summary>
 public sealed class Board
 {
+    /// <summary>
+    /// List of White Pieces
+    /// </summary>
+    public Piece[] White { get; private set; } = new Piece[5];
+
+    /// <summary>
+    /// List of Black Pieces
+    /// </summary>
+    public Piece[] Black { get; private set; } = new Piece[5];
```

Comment:
Tu peux mettre ces deux propriétés en `readonly`, vu que les tableaux (et non pas les pièces) ne seront jamais réaffectés
```suggestion
    /// <summary>
    /// List of White Pieces
    /// </summary>
    public readonly Piece[] White = new Piece[5];

    /// <summary>
    /// List of Black Pieces
    /// </summary>
    public readonly Piece[] Black = new Piece[5];
```

Reply:
J'ai essayé mais ça me mettait une erreur

Reply:
On verra ça en Live Share